frontend/src/components/WhyPerflow/WhyPerflow.css

```diff
@@ -0,0 +1,118 @@
+#root{
+}
+
+h1.Title {
+    margin-bottom: 10px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    color: black;
+  }
+    
+    p.part2, p.part3 {
+      font-size: 17px;
+    margin-right: auto;
+    margin-bottom: 0px;
+
+    } 
+
+    h3.part1, h4.card-title{
+        color:black;
+    }
+
+    #card1column, #card2column{
+
+    }
+   
+    #title1, #title2{
+      font-size: 80px;
+      background-image: url("/perflowShorterlowsat.png");
```

Comment:
Nice job here! It’s really nice to have all the code in their corresponding place! 👍